frappe/core/doctype/background_task/background_task.json

```diff
@@ -0,0 +1,161 @@
+{
+ "actions": [],
+ "allow_rename": 1,
+ "beta": 1,
+ "creation": "2024-02-23 13:12:36.222147",
+ "doctype": "DocType",
+ "engine": "InnoDB",
+ "field_order": [
+  "task_id",
+  "status",
+  "user",
+  "method",
+  "queue",
+  "timeout",
+  "at_front",
+  "kwargs",
+  "result",
+  "success_callback",
+  "failure_callback",
+  "stopped_callback",
+  "original_task",
+  "task_start",
+  "task_end",
+  "task_runtime"
+ ],
+ "fields": [
+  {
+   "fieldname": "task_id",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Task ID",
+   "not_nullable": 1,
+   "read_only": 1,
+   "reqd": 1,
+   "search_index": 1
+  },
+  {
+   "fieldname": "status",
+   "fieldtype": "Select",
+   "label": "Status",
+   "options": "Queued\nIn Progress\nCompleted\nFailed\nStopped",
+   "read_only": 1,
+   "reqd": 1,
+   "search_index": 1
+  },
+  {
+   "fieldname": "user",
+   "fieldtype": "Link",
+   "label": "User",
+   "options": "User",
+   "read_only": 1,
+   "search_index": 1
+  },
+  {
+   "fieldname": "result",
+   "fieldtype": "Code",
+   "label": "Result",
+   "read_only": 1
+  },
+  {
+   "fieldname": "method",
+   "fieldtype": "Data",
+   "label": "Method",
+   "read_only": 1
+  },
+  {
+   "fieldname": "success_callback",
+   "fieldtype": "Code",
+   "label": "Success Callback",
+   "read_only": 1
+  },
+  {
+   "fieldname": "failure_callback",
+   "fieldtype": "Code",
+   "label": "Failure Callback",
+   "read_only": 1
+  },
+  {
+   "fieldname": "stopped_callback",
```

Comment:
lets remove all the callbacks. No one uses it afaik. Also it's elegant to handle this stuff in job itself.